### ISSUE TITLE
Add control plane calls to snifgen or modify it

### ISSUE THREAD

**Issue by ibaldin** [JeffersonLab/E2SAR#36]
It would be convenient to grow snifgen into a full fledged self-contained send/receive client that can register itself, do add senders, send sync packets and so on. 

Prerequisite: complete python bindings

**Comment by ibaldin** [comment]
This is a won't fix. Not needed.
